docvalues.go

```diff
@@ -116,13 +118,21 @@ func (s *SegmentBase) loadFieldDocValueReader(field string,
 		fdvIter.chunkOffsets[i] = loc
 		offset += uint64(read)
 	}
-
+	s.bytesRead += offset
 	// set the data offset
 	fdvIter.dvDataLoc = fieldDvLocStart
 
 	return fdvIter, nil
 }
 
+func (di *docValueReader) BytesRead() uint64 {
```

Comment:
commentary indicating that these methods are an interface implementation and the reference to that interface here would be useful.

Reply:
done.